app/src/components/map/CustomAreaButtons.vue

```diff
@@ -78,29 +78,7 @@
         <span>Clear selection</span>
       </v-tooltip>
     </v-card>
-    <v-card
-    v-if="customChartButtonVisible">
-        <v-tooltip left>
-            <template v-slot:activator="{ on }">
-            <div v-on="on" class="d-inline-block">
-                <v-btn
-                color="white"
-                fab
-                class="pa-0"
-                :style="`${$vuetify.breakpoint.mdAndDown
-                    ? 'width: 36px; height: 36px;'
-                    : 'width: 30px; height: 30px;'}
-                    border-radius: 4px;
-                    color: ${appConfig.branding.primaryColor};`"
-                @click="fetchCustomAreaIndicator"
-                >
-                <v-icon>mdi-poll</v-icon>
-                </v-btn>
-            </div>
-            </template>
-            <span>Draw chart from sub-area</span>
-        </v-tooltip>
-    </v-card>
+
```

Comment:
thanks for removing this, then also `customChartButtonVisible` and `fetchCustomAreaIndicator` can be removed in this file